src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -100,14 +96,21 @@ g.test('bindingTypeSpecific_optional_members')
 
     let success = true;
     if (!(type in kBufferBindingTypeInfo)) {
+      if (hasDynamicOffset !== undefined) success = false;
       if (minBufferBindingSize !== undefined) success = false;
     }
     if (!(type in kTextureBindingTypeInfo)) {
       if (viewDimension !== undefined) success = false;
+      if (
+        kBindingTypeInfo[type].resource === 'storageTex' &&
+        (viewDimension === 'cube' || viewDimension === 'cube-array')
```

Comment:
Put this inside the `if (kBindingTypeInfo[type].resource !== 'storageTex')` on line 115 instead?

Reply:
Done.